simple_form.gemspec

```diff
@@ -1,25 +1,24 @@
-# -*- encoding: utf-8 -*-
-$:.push File.expand_path("../lib", __FILE__)
-require "simple_form/version"
+$LOAD_PATH.push File.expand_path('../lib', __FILE__)
+require 'simple_form/version'
 
 Gem::Specification.new do |s|
-  s.name        = "simple_form"
+  s.name        = 'simple_form'
```

Comment:
Can you avoid all those changes in the gemspec that are not to add support to Rails 5.2?

Reply:
Sure. I just used Rubocops autoformat here. Will revert the changes.

Reply:
@rafaelfranca done.